libs/service-portal/settings/access-control/src/index.ts

```diff
@@ -12,11 +12,16 @@ export const accessControlModule: ServicePortalModule = {
   name: 'Aðgangsstýring',
   widgets: () => [],
   routes: ({ userInfo }) => {
+    const isCompany = userInfo.profile.subjectType === 'legalEntity'
+    const isDelegation = Boolean(userInfo.profile.actor)
```

Comment:
These checks are done more than once and are not excactly the same. Would recommend creating a helper that ensures consistency, eases refactoring, and allows for a centralized place for documenting behavior (i.e. it's not obvious why `isDelegation = Boolean(userInfo.profile.actor)`)

Reply:
Agree 🚀 Will refactor in another PR